docs/packages/nest/overview.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Overview
+label: NestjS
+---
+
+import SharedOverview from '../node/\_shared-overview.md';
+
+# Platformatic NestJS
+
+The Platformatic NestJS allows to run a [NestJS](https://nestjs.com/) application as a Platformatic Runtime service with no modifications.
+
+## Getting Started
+
+Create or copy a NestJS application inside the `web` or `services` folder. If you are not using [`autoload`](../../runtime/configuration.md#autoload), you also have to explictly add the new service.
+
+You are all set, you can now start your runtime as usual via `wattpm dev` or `wattpm start`.
+
+## Install
+
+```bash
+npm install @platformatic/nest
+```
+
+## Example configuration file
+
+```json
+{
+  "$schema": "https://schemas.platformatic.dev/@platformatic/nest/2.66.0.json",
+  "application": {
+    "basePath": "/frontend"
+  }
+}
+```
```

Comment:
If we do that, how does a developer can get started with this?

Reply:
Ok, I've updated the documentation specifying what those example are supposed to be put in.